documentation/docs/concepts/decentralized_identifiers/create.mdx

```diff
@@ -25,17 +25,28 @@ Select your programming language of choice and press the green play button to ex
 
 :::
 
-<CodeSnippet 
-    nodeReplitLink="https://repl.it/@IOTAFoundation/create-did?lite=true"
+<CodeSnippet
+    nodeReplitLink="https://repl.it/@IOTAFoundation/Create-DID-07?lite=true"
     rustContent={createDidRustExample}
     nodeGithubLink = "https://github.com/iotaledger/identity.rs/blob/dev/bindings/wasm/examples-account/src/create_did.ts"
-    rustGithubLink = "https://github.com/iotaledger/identity.rs/blob/dev/examples_legacy/account/create_did.rs"
+    rustGithubLink = "https://github.com/iotaledger/identity.rs/blob/dev/identity_stardust/examples/ex0_create_did.rs"
 />
 
-The first step in this example is the creation of an account. This acts as a stateful object that manages one or more identities. The account provides an interface to execute high-level operations on identities, such as creating, updating, and storing them.
+This examples creates a new address with funds using the faucet on the testnet. Next, it creates a new DID Document with a verification method. This DID Document is then published in an Alias Output making the DID available in the ledger state and can be resolved by any node.
+
+Note that using the faucet is only possible on the testnet. Once the Stardust update is released on the Shimmer network or the IOTA network, an output with funds is required to create a new Alias Output that represents a DID.
+
+Updating the identity is possible, which will be discussed in the next section, but all previous states of the DID Document will still be stored in permanodes and any third party recording the network activity.
```

Comment:
```suggestion
Updating the identity is also possible, which will be discussed in the next section, but all previous states of the DID Document will still be stored in permanodes and can be read by any third party recording the network activity.
```

Reply:
What is meant here is that a third party can store the transactions of the network, but it's not necessarily a permanode.

Reply:
```suggestion
Updating the identity is also possible, which will be discussed in the next section, but all previous states of the DID Document will still be stored in permanodes or any third party recording the network activity.
```

Reply:
Makes sense, updated the suggestion accordingly.